<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Game</class>
 <widget class="QWidget" name="Game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>斗地主</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="meta">
     <item>
      <widget class="QLabel" name="upper">
       <property name="font">
        <font>
         <family>DengXian</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dipai">
       <property name="font">
        <font>
         <family>DengXian</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lower">
       <property name="font">
        <font>
         <family>DengXian</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="spacerX">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="messages">
     <item>
      <widget class="QLabel" name="upperMessage">
       <property name="font">
        <font>
         <family>DengXian</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="myMessage">
       <property name="font">
        <font>
         <family>DengXian</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lowerMessage">
       <property name="font">
        <font>
         <family>DengXian</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="spacerY">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="actionContainer">
     <item>
      <widget class="QPushButton" name="negativeAction">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>DengXian</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>不要</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="positiveAction">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>DengXian</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>出牌</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="cardContainer"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
